jbpm-installer/src/test/java/org/jbpm/persistence/scripts/TestPersistenceContext.java

```diff
@@ -117,33 +127,30 @@ public void clean() {
      * Executes SQL scripts from specified root SQL scripts folder. Selects appropriate scripts from root folder
      * by using dialect that is defined in datasource.properties file.
      * @param scriptsRootFolder Root folder containing folders with SQL scripts for all supported database systems.
+     * @param dropFiles indicates if dropFiles are excluded or not
      * @throws IOException
      */
-    public void executeScripts(final File scriptsRootFolder) throws IOException, SQLException {
-        executeScripts(scriptsRootFolder, null);
+    public void executeScripts(final File scriptsRootFolder, boolean dropFiles) throws IOException, SQLException {
+        executeScripts(scriptsRootFolder, dropFiles, null);
     }
     
-    public void executeScripts(final File scriptsRootFolder, String type) throws IOException, SQLException {
+    public void executeScripts(final File scriptsRootFolder, boolean dropFiles, String type) throws IOException, SQLException {
```

Comment:
Now I think if I set dropFiles to true, it will actually execute creation scripts, is that right? Shouldn't it be the other way around?

Reply:
Renamed to "createFiles"